### ISSUE TITLE
0.3 doesn't work on Android Native Browser

### ISSUE THREAD

**Issue by newjing** [addyosmani/basket.js#46]
tested on Android 2.3.4

0.2 works well, while 0.3 doesn't work

**Comment by sindresorhus** [comment]
Can you paste in the output in the Android console or something?

Is it your own project it doesn't work with or the demo on the website?

**Comment by felipemorais** [comment]
Using android 4.0.4 
0.2 and 0.3 works fine.

**Comment by addyosmani** [comment]
Works fine with latest for me.

**Comment by newjing** [comment]
To avoid basket.js not supported in some case, I did a test after basket.js is added

```
    if (typeof(basket) == "undefined" || basket == null) {
               // quote js by coding
    }else {
               //other logics
    }
```

with 0.2 , this works fine; 
with 0.3 , it works on PC chrome, but not on native browser of Android 2.3.4

I don't know how I can debug in mobile browser on a real phone.  Sth wrong in my testing code?

**Comment by peol** [comment]
I have an old HTC Desire, charging it now and I'll have a swing at this.

**Comment by peol** [comment]
I have a pending fix, should I run `grunt` and commit the `dist/` files as well?

@newjing checking for `window.basket` should be enough, it's an object on my Android phone (2.3.6) so there shouldn't be any problem detecting it. Though, `basket.get()` is broken in Android < 3.0 (that's what the pending fix is for) so try again when that's been committed later today.

**Comment by sindresorhus** [comment]
> I have a pending fix, should I run grunt and commit the dist/ files as well?

No, only on releases.

**Comment by peol** [comment]
> No, only on releases.

Alright, thought we were announcing on wednesday so this would be included in the 0.3.0 tag (or 0.3.1). I'll push it now then, all unit tests passed on Android 2.3.6.

**Comment by sindresorhus** [comment]
@peol I think the easiest is to just do compile it in the same commit that bumps the package.json and component.json. I dunno, we might do more changes before wednesday.

Thanks for the fix btw :) Always a pleasure having to fix Android <4.0 bugs :p

**Comment by peol** [comment]
@sindresorhus Agreed, makes sense :)
